core/client/fs/src/test/java/alluxio/client/file/cache/store/PageStoreTest.java

```diff
@@ -101,6 +112,25 @@ public void getOffsetOverflow() throws Exception {
     }
   }
 
+  @Test
+  public void getPages() throws Exception {
+    mOptions.setRootDir(mTemp.getRoot().getAbsolutePath());
+    int len = 32;
+    int count = 16;
+    byte[] data = BufferUtils.getIncreasingByteArray(len);
+    Set<PageId> pages = new HashSet<>(count);
+    try (PageStore store = PageStore.create(mOptions)) {
+      for (int i = 0; i < count; i++) {
+        PageId id = new PageId(0, i);
+        store.put(id, data);
```

Comment:
all page values are the same, can we use different page content here?

Reply:
This test only focus on verifying the page ids. The content is verified in other tests.